10/umbraco-commerce/installation/licensing-model.md

```diff
@@ -67,6 +67,14 @@ Once you have received your license code it needs to be installed on your site.
 }
 ```
 
+{% hint style="info" %}
+If using environment variables rather than a JSON file for configuration you may run into issues with the periods used in the product key.  If so, an underscore can be used instead:
```

Comment:
Could this, an potentially the rest, be stripped down to this? Or is it lacking too much detail? 🤔 

```suggestion
Use an underscore in the product name, to avoid issues when using environment variables.
```

Reply:
It would be OK, but the detail is kind of useful too for background, and also as we've documented above using the ".".

Reply:
Yeah, I see 🤔 

How about:

"When using environment variables you might run into issues when using a period in the product name. Use an underscore in the product name instead, to avoid issues."

Reply:
Sounds fine, do you want me to make that update or will you do so?